python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -604,6 +607,14 @@ class RandomForestClassifier(Base):
             When set to True, the predict method will, when necessary, convert
             the input to the data type which was used to train the model. This
             will increase memory used for the method.
+        storage_type : string (default = 'SPARSE')
```

Comment:
storage_type parameter name is very confusing and counter intuitive the way It is named, seems to imply this is the format for saving something to a file. A variable name like sparse would me much better. Also the parameter names are very unpythonic, the capital letter strings are bad. Accepting the three values True, False and ‘auto’ is what I would propose.

Reply:
I agree with storage_type is confusing. How about tree_format="sparse"/"dense"

Reply:
I updated it to `sparse_forest = True/False/auto` but if everyone prefers one name and possible values ove the other I can edit the files again

Reply:
I think both names could work. I like the fact that `sparse_forest` can lead to a binary parameter with an extra option of auto, but `tree_format` is not bad. @JohnZed opinion would be nice to have when he's back